Gems/ScriptCanvas/Code/Builder/ScriptCanvasBuilderWorker.h

```diff
@@ -68,7 +68,7 @@ namespace ScriptCanvasBuilder
         ExecutionStateAsLightUserdata,
         UpdateDependencyHandling,
         AddExplicitDestructCallForMemberVariables,
-
+        DoNotLoadScriptEventsDuringCreateJobs,
```

Comment:
Nit: Maybe change it to something like "DoNotLoadReferencedAssetsDuringCreateJobs"? I assume that other type resources are loaded as well without this change.